src/GraphQL.php

```diff
@@ -379,7 +379,7 @@ protected function parseQuery(string $query): DocumentNode
      *
      * @param  array<string, \GraphQL\Validator\Rules\ValidationRule>  $validationRules
      *
-     * @return list<\GraphQL\Error\Error>
+     * @return array<int, \GraphQL\Error\Error>
```

Comment:
I would rather promise more, since i believe it to be true.

Reply:
Reverted. Added exclusion to neon for `Method Nuwave\Lighthouse\GraphQL::validateCacheableRules() should return list<GraphQL\Error\Error> but returns a .*`